src/services/Item/index.ts

```diff
@@ -0,0 +1,25 @@
+import Axios from 'axios';
+import { IncomingMessage } from 'http';
+
+import { listConfig, actionConfig } from './config';
+import { ItemListParams, ItemActionRequestParams } from '@src/types';
+import { IItem } from '@src/interfaces';
+
+const list = async (
+  params: ItemListParams & { userId?: number },
+  req?: IncomingMessage
+): Promise<IItem[]> => Axios(listConfig(params, req)).then((res) => res.data);
+
+const action = async (
+  ids: number[],
+  body: ItemActionRequestParams,
```

Comment:
```suggestion
  body: ItemActionRequestParams,
```

type이름이 ItemActionRequestParams네요 ㅠㅠ 적절히 변경해주세요

Reply:
e418508